ban/tests/http/test_diff.py

```diff
@@ -20,7 +18,7 @@ def test_diff_endpoint(client):
     street.increment_version()
     street.save()
     resp = client.get('/diff')
-    assert resp.status == falcon.HTTP_200
+    assert resp.status_code == 200
```

Comment:
The [status lib](https://github.com/avinassh/status) will provide more meaningful statuses, at least for tests if you don't want an additional dependency for the core.